src/v2/ion/uiSchemaLabelTransformer.js

```diff
@@ -0,0 +1,172 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+///////////////////////////////////////////////////////////////////////////////
+//                                  How this transformer work?               //
+// 1. Figure out Path for all Labels (for normal field, or options) or Placeholder (for checkbox),e.g.
+//   - `identify.identifier`, this is the label for user name at identify page
+//   - `select-authenticator-enroll.authenticator.email`,
+//      this is for label for email authenticator at `select-authenticator-enroll` page
+// 2. Define a mapping between the path and i18n key. see `I18N_OVERRIDE_MAPPINGS`
+//   - Not all path needs i18n override. e.g.
+//     `challenge-authenticator.security_question.credentials.questionKey`,
+//     which is security question during verify step and apparently it does not
+//     need overwrite.
+//   - This mapping is subject to change when API starts sending i18n key along with label.
+//     It probably becomes a mapping between API's i18n key and key in login.properties
+//     in order to be backward compatible.
+// 3. Now find i18n value using such route: `i18n Path -> i18n key -> login.properties`
+//    If found an i18n value, replace label by this i18n value.
+//
+///////////////////////////////////////////////////////////////////////////////
+
+import { _, loc, $ } from 'okta';
+import Bundles from 'util/Bundles';
+import Logger from 'util/Logger';
+
+const SECURITY_QUESTION_PREFIX = 'enroll-authenticator.security_question.credentials.questionKey.';
+
+const I18N_OVERRIDE_MAPPINGS = {
+  'identify.identifier': 'primaryauth.username.placeholder',
```

Comment:
This mapping seems like it allows us to keep our existing set of new keys that we created for M1 under the `oie` name space i.e 
```
{
i18npath1 : oie.key1,
i18npath2 : oie.key2
}
```
What is our plan to for when we want to make existing customer overrides work?  Would we need to map those keys to the same i18n path ?

Reply:
My current thinking is to just re-use v1 keys as much as possible.

Reply:
just throwing it out there 

Maybe we can have the i18n path map to an array of keys which can have both M1 and V1 keys. This way we can try to handle the override case for v1 customers.

```
'identify.identifier': ['oie.username','primaryauth.username.placeholder']
```

Reply:
The way we can support backward compatible for V1 customer only on those label that from API.
We still lacking the ability of backward compatible for those new keys used directly in v2 views. 
Maybe need to use this mechanism anywhere need backward compatible?